doc/source/using_config.rst

```diff
@@ -710,33 +712,46 @@ The ``auth`` configuration block looks like this:
      server-cert: server.crt
      client-cert: client.crt
      client-key: client.key
+     access-token: access.token
+     access-token-reload-interval: 60
 
 
 Attributes
 ''''''''''
 
 * ``server-cert``
 
-  The server certificate is used to decrypt traffic coming from the
-  server.
+  The server certificate is used to verify the identity of the server instead
+  of using the CA store from the operating system for verification.
 
 * ``client-cert``
 
-  The client certificate is used by the remote server to decrypt
-  traffic being uploaded to the server.
+  The client certificate is used by the remote server to authenticate the client.
 
   The remote server will have its own copy of this certificate, but the
   client needs to send this certificate's identity to the server so that
   the server knows which certificate to use.
```

Comment:
Thanks for fixing the descriptions. I believe this paragraph is still slightly incorrect. The server doesn't need to have a copy of the certificate if it is signed by a CA that the server trusts.

Reply:
True, a CA generally also makes sense for client certificates (although I would expect that to be an internal CA). I don't think these authentication details on the server side are actually relevant or helpful for client configuration, so I've removed that paragraph.